### PR TITLE
Add Ocr2 Proxy E2E Test

### PR DESCRIPTION
**Pull request by tateexon**
Also bumped up the concurrency from 2 to 3 so we hopefully don't increase run time in ci by much if any.

### CODE REVIEW

File: tests/e2e/smoke/ocr2_proxy_test.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	tc "github.com/smartcontractkit/chainlink-terra/tests/e2e/smoke/common"
+	"github.com/smartcontractkit/integrations-framework/actions"
+)
+
+var _ = Describe("Terra OCRv2 Proxy @ocr_proxy", func() {
+	var state *tc.OCRv2State
+
+	BeforeEach(func() {
+		state = &tc.OCRv2State{}
+		By("Deoloying the cluster", func() {
```
Comment (aalu1418) [inline tests/e2e/smoke/ocr2_proxy_test.go:19]:
```suggestion
		By("Deploying the cluster", func() {
```

**Review by aalu1418 — APPROVED** [review]
LGTM! (just a spelling change that i noticed)

File: tests/e2e/smoke/ocr2_proxy_test.go (hunk 1 above)
Comment (tateexon) [reply tests/e2e/smoke/ocr2_proxy_test.go]: Good catch! I copy pasted from another test and it had the issue so I fixed it in all the copy pasted areas and found some others.

**Review by aalu1418 — APPROVED** [review]
🎉

**Pull request merged**